pkg/driver/common/operator/hooks_test.go

```diff
@@ -146,6 +146,55 @@ func Test_WithHyperShiftNodeSelector(t *testing.T) {
 	}
 }
 
+func Test_WithHyperShiftLabels(t *testing.T) {
+	tests := []struct {
+		name           string
+		hcp            *hypev1beta1api.HostedControlPlane
+		expectedLabels map[string]string
+	}{
+		{
+			name:           "no labels",
+			hcp:            getTestHostedControlPlane("hcp_no_labels.yaml"),
+			expectedLabels: nil,
+		},
+		{
```

Comment:
Please add a test that existing labels are not replaced

Reply:
added